src/Host.cpp

```diff
@@ -2017,7 +2023,19 @@ bool Host::uninstallPackage(const QString& packageName, int module)
 
     QString dest = mudlet::getMudletPath(mudlet::profilePackagePath, getName(), packageName);
     removeDir(dest, dest);
-    saveProfile();
+
+    static std::optional<bool> saveTimer;
```

Comment:
Why a `static`  local in this method and not a `private` class member?

Reply:
It's only ever needed by this method, so I kept it local here to keep the complexity of the class overall down.

Reply:
I am confused, after trying to research this, https://stackoverflow.com/q/3778450/4805858 , is there a separate `saveTimer` for *each* profile and each of them saves their state between calls of this method in that profile OR is there a single, unique `saveTimer` that is common to ALL instances of the `Host` class?

:confused:

Reply:
Each profile... so you pointed out a good problem! I will change the timer to be from all profiles to per-profile.

Reply:
That is just it though - I thought it was *per profile* - but the answers to that SO seem to suggest the opposite i.e. one single thing shared between every instance of the class with that method. {I hope I haven't got that wrong in any of my prior code...}

Something that coming from **C**-coding didn't cover. :confounded:

Reply:
Fixed.